### DISCUSSION TITLE
Links do not work anymore

### DISCUSSION THREAD

**Discussion by th3-sh0w3r** [afconsult/quill-mention#354] [Q&A]
Since some unknown time the links do not work anymore.
I didn't change any of the code, just updated the package.
I'm giving the link as a data parameter and it also links that with data-link to the span element, but there is no <a> tag and it is also not clickable anymore.

**Comment by csculley** [comment]
Did you update from <v4 to after v4? The [upgrade notes](https://github.com/quill-mention/quill-mention#upgrade-notes) have instructions on how to avoid XSS vulnerabilities that you would have from saving HTML elements in the delta

**Reply by th3-sh0w3r** [reply]
I have version ^4.0.0
I don't pass any HTML data or XSS sensitive data. Only text.
![2023-11-08 10_13_20-Evalea - Wiki – Opera](https://github.com/quill-mention/quill-mention/assets/13116462/b28cb1a3-3bc2-482e-b666-e1f64af25ff0)
![2023-11-08 10_14_49-Evalea - Wiki – Opera](https://github.com/quill-mention/quill-mention/assets/13116462/6004c545-1f3b-4428-9b20-a0821637d476)
